src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/scanOrchestrator.hpp

```diff
@@ -110,6 +127,14 @@ class TScanOrchestrator final
             {
                 m_integrityClearOrchestration->handleRequest(std::move(context));
             }
+            else if (type == ScannerType::DatabaseUpdate)
```

Comment:
We should review the approach.
The scan of all packages of all agents only will be triggered by the module itself when it realizes it has been disabled for some time, it won't arrive as an external event. 
It'll also take too much time so it should run in a separe thread.

Reply:
Done in new refactor. We can discuss this again if further changes need to be implemented